bcftools.wdl

```diff
@@ -76,14 +143,13 @@ task Annotate {
         ~{"--samples-file " + samplesFile} \
         ~{true="--single-overlaps" false="" singleOverlaps} \
         ~{true="--remove" false="" length(removeAnns) > 0} ~{sep="," removeAnns} \
+        ~{"--threads " + threads} \
         ~{inputFile}
-        bcftools index --tbi ~{outputPath}
```

Comment:
Is there a specific reason why indexing is being removed here? It still produces a vcf.gz so indexing shouldn't be an issue.